docs/getting_started/requirements.md

```diff
@@ -18,34 +18,34 @@ These setups are tested by QA and are generally recommended setups. For security
+|Cluster PHP packages|[php-redis](https://pecl.php.net/package/redis) *(3.1+)*|[php-redis](https://pecl.php.net/package/redis) *(3.1+)*|[php-redis](https://pecl.php.net/package/redis) *(3.1+)*|
 
 |||
 |------|------|
-|Search|Solr (recommended; for performance, features and search quality):</br></br>Solr 6.x *Currently tested with Solr 6.6LTS*</br></br>Oracle Java/Open JDK: 8</br></br>*NOTE: If your Java version is higher than 8, you need to downgrade it to Oracle Java/Open JDK: 8.* |
+|Search|Solr (recommended; for performance, features and search quality):</br></br>Solr 7.x *Currently tested with Solr 7.7LTS*</br></br>Oracle Java/Open JDK: 8 or higher |
 |Graphic Handler|GraphicsMagick or ImageMagick or GD|
-|[Clustering](../guide/clustering.md)|Linux NFS *or* S3/EFS *(for IO, aka binary files stored in content repository, not supported with legacy)*</br>Redis 3.2 or higher *(preferably separate instances for session & cache, both using one of the `volatile-*` [eviction policies](https://redis.io/topics/lru-cache))*</br>[Varnish](http://varnish-cache.org/) 5.1 or 6.0LTS *(recommended)* with [varnish-modules](https://github.com/varnish/varnish-modules/blob/master/README.rst) *or* [Fastly](https://www.fastly.com/) using [our bundle provided with eZ Platform Enterprise](../guide/http_cache.md#serving-varnish-through-fastly) *(for HttpCache)*|
+|[Clustering](../guide/clustering.md)|Linux NFS *or* S3/EFS *(for IO, aka binary files stored in content repository, not supported with legacy)*</br>Redis 3.1 or higher *(preferably separate instances for session & cache, both using one of the `volatile-*` [eviction policies](https://redis.io/topics/lru-cache))*</br>[Varnish](http://varnish-cache.org/) 6.0LTS with [varnish-modules](https://github.com/varnish/varnish-modules/blob/master/README.rst) *or* [Fastly](https://www.fastly.com/) using [our bundle provided with eZ Platform Enterprise](../guide/http_cache.md#serving-varnish-through-fastly) *(for HttpCache)*|
 |Filesystem|Linux ext4 / XFS|
 |Package manager|Composer (recent stable version)|
 |Asset manager|`Node.js` 10.15.3 LTS</br>`yarn` 1.15.2 or higher|
 
 ### Other supported setups
 
-For security and performance we generally recommend (unless otherwise noted and marked with \*) using the newer versions of components below.
+For security and performance we generally recommend (unless otherwise noted) using the newer versions of components below.
 
 -   OS: Linux
 -   Web Servers:
     -   Apache 2.4, with required modules `mod_rewrite`, `mod_env` and recommended: `mod_setenvif`, `mod_expires`
         - event MPM is recommended, if you need to use _prefork_ you'll also need the `mod_php` module
-    -   Nginx 1.12, 1.14
+    -   Nginx 1.12, 1.14, 1.16
 -   DBMS
-    -   MySQL 5.7\* or 8.0\* \**
-    -   MariaDB 10.0, 10.1, 10.2\*, 10.3\*
+    -   MySQL 5.7 or 8.0
+    -   MariaDB 10.2, 10.3, 10.4
```

Comment:
NOTE: I don't really know if this is correct, the issues someone experienced with 10.1 was rather due to mysql settings and usage of default empty string value. So would be good to know what @alongosz thinks here, with a grain of salt.

Reply:
Fine by me as long as you're bumping, not downgrading ;)

Reply:
ok :)